ctapipe/image/cleaning.py

```diff
@@ -2,16 +2,65 @@
 Image Cleaning Algorithms (identification of noisy pixels)
 """
 
-__all__ = ['tailcuts_clean', 'dilate']
+__all__ = ['tailcuts_clean', 'dilate', 'fact_image_cleaning', 'ImageCleaning']
 
 import numpy as np
 from scipy.sparse.csgraph import connected_components
-
-
-def tailcuts_clean(geom, image, picture_thresh=7, boundary_thresh=5,
-                   keep_isolated_pixels=False,
-                   min_number_picture_neighbors=0):
-
+from ctapipe.core import Component
+from traitlets import Dict, CaselessStrEnum
+
+
+class ImageCleaning(Component):
```

Comment:
Rename this to `ImageCleaner` so that we are using the same grammar as other objects in ctapipe (Reconstructor, ChargeExtracter, etc).  "Image Cleaning" is a general concept of denoising, and "Image Cleaner" is something specifically that does denoising.